Rakefile

```diff
@@ -65,9 +65,7 @@ task :generate_parser_caches => [PARSER_CACHE_DIR] do
 
     cache = YAML.load_file(filename.to_s)
     cache[:files] = cache[:files].delete_if do |k, _|
-      k =~ /\Aforeman/ &&
-        k !~ /\Aforeman_proxy_content/ &&
-        k !~ /\Aforeman_proxy_certs/
+      !%w(certs foreman_proxy_content katello puppet).include?(k)
```

Comment:
Yes it was the whole cause of the bug initially was puppet being in there, I have no idea why I included in the list :man_facepalming: